src/config.cpp

```diff
@@ -130,9 +146,32 @@ QMap<QString, QString> PorymapConfig::getKeyValueMap() {
     map.insert("recent_map", this->recentMap);
     map.insert("pretty_cursors", this->prettyCursors ? "1" : "0");
     map.insert("map_sort_order", mapSortOrderMap.value(this->mapSortOrder));
+    map.insert("restore_geometry", this->restoreGeometry ? "1" : "0");
```

Comment:
I don't think having this `restore_geometry` boolean is necessary to save.  I don't think there is a case where you'd wan to disable it. (and there is no place in the UI to disable it)